.github/workflows/marimo-bot.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         env:
           HATCH_INDEX_USER: ${{ secrets.TEST_PYPI_USER }}
           HATCH_INDEX_AUTH: ${{ secrets.TEST_PYPI_PASSWORD }}
-        run: hatch publish --repo test || true
```

Comment:
i added this as a way to support `--skip-existing`, but hatch returns a successful exist code if it already exists